packages/universal-components/src/RangeDatePicker/mocks/monthsWithDistributedWeeks.js

```diff
@@ -0,0 +1,147 @@
+// @flow
```

Comment:
I don't think this is a mock, this is a dataset.

Reply:
Renamed